connector/manifest.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
-<connector-plugin class='mongosql_jdbc' superclass='jdbc' plugin-version='0.0.0' name='MongoSQL' version='18.1' min-version-tableau='2020.4'>
+<connector-plugin class='mongosql_jdbc' superclass='jdbc' plugin-version='0.1.0' name='MongoSQL' version='18.1' min-version-tableau='2021.1'>
   <vendor-information>
       <company name="MongoDB"/>
-      <support-link url="https://example.com"/>
-      <driver-download-link url="https://drivers.example.com"/>
+      <support-link url="https://support.mongodb.com"/>
+      <driver-download-link url="https://search.maven.org/search?q=a:mongodb-jdbc"/>
   </vendor-information>
   <connection-customization class="mongosql_jdbc" enabled="true" version="10.0">
-    <vendor name="vendor"/>
-    <driver name="driver"/>
+    <vendor name="MongoDB"/>
+    <driver name="MongoSQL Connector"/>
```

Comment:
do you know where these values actually get used by the connector? I couldn't find any explanation in the docs

Reply:
I don't see where it is used either, it doesn't show up in the logs.  The fields are needed to pass the packager validation however.

Reply:
Weird, their doc says "Vendor defined attributes will be logged and persisted to Tableau workbook xml in plain text." https://tableau.github.io/connector-plugin-sdk/docs/connection-dialog
It also seems that they can be used as connection attributes, but I don't think we need them, do we?

Reply:
The vendor defined attributes referred to there are the ones defined in the connectionFields.xml that start with the `v-` like the `v-loglevel` for example.
The fields in connectionFields.xml are the ones that would be used for the `<required-attributes>` element, which we wouldn't need if we use the inferred connection resolver, added in the 2021.1 release.

Reply:
I see.
Well, then it will stay a mystery. It could also be something related to a Tableau server user case. Who knows...